vars/util.groovy

```diff
@@ -671,4 +671,25 @@ String displayDurationFromSeconds(int durationInSec) {
     }
     result += "${seconds}s"
     return result
+}
+
+def umbToGHPRB(String umbMessage, String repositoryName, String pullRequestId) {
+    if(umbMessage?.trim()) {
+        def ghrprbToGHAPIPRMap = [
+            'ghprbSourceBranch' : 'head.ref',
+            'ghprbTargetBranch' : 'base.ref'
+        ] // TODO: define the rest of the map elements
```

Comment:
@Ginxo only need to know if these are 17 elements (all ghprb params) or only this five  
ghprbPullId
ghprbPullLink
ghprbGhRepository
ghprbSourceBranch
ghprbTargetBranch